### DISCUSSION TITLE
Where are the Hydrogen User-Drumkits?

### DISCUSSION THREAD

**Discussion by nathanaelmeister** [hydrogen-music/hydrogen#1675] [General]
I used to create quite a lot of projects with the hydrogen software. For those projects I have been using a big range of user-drumkits that have been available for download through the hydrogen user interface.

Some years later it does not seem to be possible to download this drumkits (by default) through this version shipped with the recent debian testing distribution.

```
Operating System: Debian GNU/Linux bookworm/sid   
          Kernel: Linux 6.0.0-3-amd64
    Architecture: x86-64
```

`hydrogen 1.1.1-`

The address for the server list supplied here is: http://freepats.zenvoid.org/hydrogen.xml 

![image](https://user-images.githubusercontent.com/29178298/201779667-410b0682-5462-49a2-a697-b22ab71dace3.png)

Though this seems to be a pretty small list, with none of the drumkits used to be available for download. Is there a different server addresses that I can add to the list?

**Comment by theGreatWhiteShark** [comment]
Hey @nathanaelmeister 

> The address for the server list supplied here is: http://freepats.zenvoid.org/hydrogen.xml

Thanks for link! I wasn't aware of these kits. And I didn't know that there are actual kits out there using the drumkit image feature :D

I just tried downloading them on `master` (with a version of Hydrogen soon to be released as 1.2.0-beta) and everything worked. Could you check whether the latest code works for you too?
